src/pageContainer/productPg/productPg.css

```diff
@@ -0,0 +1,85 @@
+.productPg {
+  width: 100%;
+}
+.baseContainer {
+  overflow: scroll;
+  height: 100%;
+  width: 100%;
+}
+.page-content {
+  height: 100%;
+  width: 100%;
+}
+.product-content {
+  display: flex;
+  height: 100%;
+  width: 100%;
+}
+
+.sideNav {
+  flex-grow: 3;
+  position: sticky;
+  position: -webkit-sticky;
+  position: -moz-sticky;
+  position: -ms-sticky;
+  position: -o-sticky;
+  width: 30%;
+  min-width: 9rem;
+  max-width: 20rem;
+  background: var(--stdWhite-color);
+  border: 1px solid black;
+  padding: 0.5rem;
+  top: 0;
+}
+.main-content {
+  flex-grow: 6;
+  display: flex;
+  flex-direction: column;
+  padding: 2.25rem;
+  max-height: 100%;
+  overflow: scroll;
+}
+.sideNav-header {
+  display: flex;
```

Comment:
I'm seeing multiple use of Flex, how about creating class named `flex` with the same property. Not necessary but DRY principle, right? Also, it would save you some time next time for sure. 

Not just that, you can also create properties for row and column as well. Rest looks good, structure and components. Keep up!

Reply:
nice suggestion will surely apply